src/views/homepage/index.tsx

```diff
@@ -60,12 +60,11 @@ const HomePageContent = ({
 					ctaText={merchandising.hcp.ctaText}
 				/>
 			</MerchandisingSlots>
-			<LearnSection
-				imageSrc={learnSection.imageSrc}
-				heading={learnSection.heading}
-				description={learnSection.description}
-				collectionCards={learnSection.collectionCards}
-				link={learnSection.link}
+			<CertificationsSection
```

Comment:
I've swapped in the revised `CertificationsSection` for the previously used `LearnSection` component.

🔪 Have also removed the `LearnSection` component, as it is no longer used.